third_party/move/move-compiler/src/parser/syntax.rs

```diff
@@ -2796,6 +2868,28 @@ fn parse_field_annot(context: &mut Context) -> Result<(Field, Type), Box<Diagnos
     Ok((f, st))
 }
 
+/// Parse a comma list of types surrounded by parenthesis into a vector of `(Field, Type)` pairs
```

Comment:
Would be nice to also give the pseudo-grammar, keeping up with the tradition of the rest of file.

Reply:
done